Source/Engine/Core/Math/Int4.h

```diff
@@ -78,16 +89,261 @@ API_STRUCT(InBuild) struct FLAXENGINE_API Int4
     {
     }
 
+    // Init
+    // @param v Int2 to use X and Y components
+    // @param z Z component value
+    // @param w W component value
+    Int4(const Int2& xy, float z, float w);
```

Comment:
Additional `z` and `w` should be `int32` to match the actual struct storage value.